Client/Frontend/Home/Wallpapers/v2.0/Models/WallpaperMetadata.swift

```diff
@@ -0,0 +1,11 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+/// Metadata, fetched from the server, to update wallpaper availability.
+struct WallpaperMetadata: Codable {
+    let lastUpdatedDate: String
```

Comment:
Is there a reason this is a String instead of a date? Also I suggest naming it `lastUpdated` as `date` is already part of that and therefor implies the content.